modules/platforms/dotnet/examples/Apache.Ignite.Examples/Compute/PeerLoadingExample.cs

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Apache.Ignite.Examples.Compute
+{
+    using System;
+    using Apache.Ignite.Core;
+    using Apache.Ignite.Core.Compute;
+
+    /// <summary>
+    /// Example demonstrating closure execution with peer assembly loading.
+    /// Should be run with standalone Apache Ignite.NET node.
+    /// "-assembly" command line argument is not required on remote nodes: assemblies are loaded automatically when needed.
```

Comment:
I would rephrase the doc this way:

Before executing the example start one or more standalone Apache Ignite.NET nodes not passing  "-assembly" command line argument. That argument is not required on remote nodes because the assemblies are loaded automatically whenever is needed.